spec/requests/api/v1/auth/sessions/destroy_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe 'DELETE api/v1/auth/sessions', type: :request do
+  let(:user) { create(:user) }
+  let(:headers) { auth_headers(user) }
+
+  subject do
+    delete destroy_api_v1_user_session_path, headers: headers, as: :json
+    response
+  end
+
+  context 'with correct headers' do
+    context 'when session exists' do
+      it { is_expected.to have_http_status(:no_content) }
+
+      it 'returns no authentication headers' do
```

Comment:
also check that the token is deleted, either by checking a change on the `token` count, or by testing if the old token is still valid, whatever you prefer, but we need to ensure the session was closed